CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+
+## [v0.3.2]
+
+### Changed
+
+- Auto scaling settings has been adjusted based on past experiences. Now coreDNS responds better to a request peak.
```

Comment:
Much <3 for adding the changelog but this looks wrong. Also shouldn't the newest version be at the top?

Reply:
indeed it is a typo because copy paste CHANGELOG